packages/salesforcedx-vscode-apex/package.json

```diff
@@ -34,7 +34,7 @@
     "path-exists": "3.0.0",
     "shelljs": "0.8.5",
     "vscode-extension-telemetry": "0.0.17",
-    "vscode-languageclient": "5.1.1"
+    "vscode-languageclient": "^7"
```

Comment:
Does this add some supported needed for this PR?  We're just gonna bump again to ^9 in the next PR correct?

Reply:
Version 7 was the last version w/o breaking changes. I will deal with any difs between this PR and the v9 one as they get merged.